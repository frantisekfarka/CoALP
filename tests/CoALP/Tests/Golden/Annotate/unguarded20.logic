q(d(s(Q1),Q2)):-p(d(Q1,Q2)).
p(d(t(P1),P2)):-r(d(P1,P2)).
r(d(u(U1),U2)):-v(d(U1,U2)).
v(d(w(V1),V2)):-q(d(V2,V2)).
