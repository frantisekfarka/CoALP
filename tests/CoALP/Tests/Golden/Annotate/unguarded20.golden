q(d(s(V_2), V_3), transform-func-1(V_-10)) :- p(d(V_2, V_3), V_-10).
p(d(t(V_4), V_5), transform-func-2(V_-11)) :- r(d(V_4, V_5), V_-11).
r(d(u(V_6), V_7), transform-func-3(V_-12)) :- v(d(V_6, V_7), V_-12).
v(d(w(V_8), V_9), transform-func-4(V_-13)) :- q(d(V_9, V_9), V_-13).
